pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+filterwarnings =
+    error
+    ignore:.*?(\b(pkg_resources\.declare_namespace)\b).*:DeprecationWarning
```

Comment:
What are we ignoring here?

Reply:
it's a warning about a way namespaces are declared in pymc-experimental, it's purely python convention related, so I thought that for now it shouldn't be the highest priority

Reply:
pkg_resources.declare_namespace() is throwing a warning

Reply:
Sounds good to ignore